### PR TITLE
PJFCB-11494 Adding 'Documentation requests' from documentation level

### PR DESCRIPTION
**Pull request by h-pop**

### CODE REVIEW

File: src/partials/toc.hbs [hunk 1]
```diff
@@ -18,45 +17,9 @@
           ><path
               d="m16 2.012 3 3L16.713 7.3l-3-3zM4 14v3h3l8.299-8.287-3-3zm0 6h16v2H4z"
             ></path></svg>
-          Edit this Page
-        </a>
-      {{else if (and
-        page.editUrl
-        (or env.FORCE_SHOW_EDIT_PAGE_LINK (not page.origin.private))
-      )}}
-        <a href="{{page.editUrl}}">
```
Comment (thugcee) [inline src/partials/toc.hbs:27]: I think that we will need this sooner or later and it should not be removed.

File: src/partials/toc.hbs [hunk 2]
```diff
@@ -18,45 +17,9 @@
...
-            width="24"
-          ><path
-              d="m16 2.012 3 3L16.713 7.3l-3-3zM4 14v3h3l8.299-8.287-3-3zm0 6h16v2H4z"
-            ></path></svg>
-          Edit this Page
+          Docs request
         </a>
       {{/if}}
-      {{#if page.origin.webUrl}}
-        <a href="{{page.origin.webUrl}}" title="Bitbucket">
```
Comment (thugcee) [inline src/partials/toc.hbs:40]: The same here.

File: src/partials/toc.hbs (hunk 1 above)
Comment (h-pop) [reply src/partials/toc.hbs:27]: I disagree, as we should not code in advance. This code has been commented out and should be removed. In the future, if we in fact need this code portion, we can take a look at `antora-ui-spring` official project, from where this originates.

File: src/partials/toc.hbs (hunk 2 above)
Comment (h-pop) [reply src/partials/toc.hbs:40]: The same here.

File: src/partials/toc.hbs (hunk 1 above)
Comment (dmaciaszekpsi) [reply src/partials/toc.hbs:27]: I have a similar opinion to hubert. For me commented out code is a trash, we have git history to recreate some content.

File: src/partials/toc.hbs (hunk 2 above)
Comment (dmaciaszekpsi) [reply src/partials/toc.hbs:40]: I have a similar opinion to hubert. For me commented out code is a trash, we have git history to recreate some content.

**Review by dmaciaszekpsi — APPROVED** [review]

**Pull request merged**